packages/jasmine/docs/BUILD.bazel

```diff
@@ -1,12 +1,51 @@
-load("@io_bazel_skydoc//skylark:skylark.bzl", "skylark_doc")
+# Copyright 2019 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load("@build_bazel_rules_nodejs//:defs.bzl", "nodejs_test")
+load("@io_bazel_skydoc//stardoc:stardoc.bzl", "stardoc")
+
+package(default_visibility = ["//visibility:public"])
 
 exports_files(["install.md"])
 
-skylark_doc(
+stardoc(
```

Comment:
any particular reason to change from skydoc in the same PR?

Reply:
wanted to use the same pattern for /doc /src /test for all of the packages and was simpler to have them all use stardoc; there is still one skylark_doc target left at the root